lib/librarian/http.js

```diff
@@ -35,55 +36,45 @@ var HTTP = function(host, port, ssl) {
 };
 
 // Passes the raw request and response streams to the projectController
-HTTP.prototype.raw = function(path, method, data, callback) {
-
-  if(typeof data === 'function') {
+HTTP.prototype.raw = function (path, method, data, callback) {
+  var req, options;
+  if (typeof data === 'function') {
     callback = data;
     data = null;
   }
 
-  var headers = null;
-  if(data) {
-    data = JSON.stringify(data);
-    headers = {
```

Comment:
Seems like someone probably intended for this to be added to the options. Maybe leave a `FIXME/TODO` in comments?